packages/web-components/src/components/card-section-carousel/__stories__/card-section-carousel.stories.ts

```diff
@@ -41,8 +39,8 @@ const Card = ({ copy = copyDefault, heading = headingDefault, href = hrefDefault
   </dds-card>
 `;
 
-export const Default = ({ parameters }) => {
-  const { pageSize } = parameters?.props?.Carousel ?? {};
+export const Default = () => {
+  const pageSize = null;
```

Comment:
@NSDrowned Is this still needed?

Reply:
You're right, not sure why I left that there... 🤦‍♂️